openshift_images/index.adoc

```diff
@@ -0,0 +1,60 @@
+[id="overview-of-images"]
+= Overview of images
+include::modules/common-attributes.adoc[]
+:context: overview-of-images
+
+toc::[]
+
+To know all about xref:../openshift_images/images-understand.adoc#understanding-images[containerized software], you must know the various terminologies:
+
+** Containers
+** Images
+** Image streams
+
+[discrete]
+== How you can use the Cluster Samples Operator
+
+During the initial start-up, the operator creates the default samples resource to initiate the creation of the image streams and templates.
```

Comment:
Suggestion based on IBM style (it may be written as start-up in the related assembly, so this is optional):
s/start-up/startup _or_ s/start-up/installation

Reply:
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s >> sounds good I would opt for `startup` and replace existing four instances of `start-up` with `startup`